### DISCUSSION TITLE
Installing Plugin Causes Fsleyes to crash, and then I am unable to open it again.

### DISCUSSION THREAD

**Discussion by aidanquandt** [neuropoly/axondeepseg#674] [Help needed]
Hi there, I hope you can help solve my problem. I have successfully installed ads, but am unable to add the plugin to fsleyes. I downloaded fsleyes using the following command: conda install -c conda-forge fsleyes. When I go into the virtual environment as directed in the setup instructions on the ads page and first run fsleyes, it launches properly. However, when I load plugin it takes a few seconds, and then closes fsleyes. The only way I am able to relaunch fsleyes after this is to uninstall fsleyes and then reinstall it. Any help?

**Comment by Stoyan-I-A** [comment]
Hello @Jumper0516, thank you for reaching out.
Which operating system are you using?

**Reply by aidanquandt** [reply]
I am using windows 10 and I have tried on both my laptop and desktop and experience the same issue.

**Reply by Stoyan-I-A** [reply]
@Jumper0516, Unfortunately FSLeyes is not supported on Windows. The rest of the AxonDeepSeg software will work on its own (command line and Jupyter notebooks) on your machine, you just won't have a graphical interface.

If you want to use AxonDeepSeg with FSLeyes, you have 2 options:

1.  Use a Virtual Machine and run Ubuntu on it. You can look up tutorials on how to use either VMware or Virutal Box with Ubuntu. Note that this will require a lot of disk space and RAM. Once you can run Ubuntu, just follow the instructions for the Linux installation.
2. Use the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/). There's a member of our team that used it previously. That being said, we don't have instructions on how to use it so you will have to figure it out on your own.

**Reply by aidanquandt** [reply]
Ok, thanks so much for the fast replies. I will try the Virtual Machine through Linux much appreciated :)

**Reply by mathieuboudreau** [reply]
@Jumper0516 Just out of curiosity, would you mind sharing how you heard about AxonDeepSeg, and what you're interested in using it for?
